### PR TITLE
Reconstruct LossAggregator and fix some typos in config files

### PR DESCRIPTION
**Pull request by wj1tr0y**
1. new config files brought by Gait3D have some typos
2. The old LossAggregator is a basic python class that cannot register parameters to the base_model. It is inconvenient to train models with those losses that require parameters, such as center loss.  
Therefore, it seems a better way that registers LossAggregator as an `nn.Module` and uses `nn.ModuleDict` to organize all losses.
**In this new version,**
LossAggregator can be indexed like a regular Python dictionary (keep it the same as the current version), 
but the modules it contains are properly registered and will be visible by all Module methods.
All parameters registered in losses can be accessed by the method 'self.parameters()', enabling they can be trained properly.

### CODE REVIEW

**Comment by wj1tr0y** [comment]
@darkliang

File: opengait/modeling/loss_aggregator.py [hunk 1]
```diff
@@ -18,16 +19,21 @@ class LossAggregator():
     Attributes:
         losses: A dict of losses.
     """
-
     def __init__(self, loss_cfg) -> None:
         """
-        Initialize the loss aggregator.
+        Initialize the loss aggregator.
```
Comment (darkliang) [inline opengait/modeling/loss_aggregator.py:24]: extra space

**Review by darkliang — APPROVED** [review]
